public/css/software-agency.css

```diff
@@ -0,0 +1,65 @@
+.banner-text.software-agency {
+  font-size: 70px;
+}
+.image-banner.banner-software-agency{
+  background-image: url("../img/health-toolkit-meeting.png");
+
+}
+.container.software-agency{
+  background:none;
+
+}
+.form {
+<style scoped>
```

Comment:
HTML can not be in a CSS file

Reply:
Solved in the new PR

Reply:
I think you may have mixed up some of the terminology here. 

You haven't created a new PR, you've added commits to this same PR. 

When I said (above) that you need to create a new PR, I mean it needs to be completely separate from this entire PR and put into a separate pull request. This can be a little tricky to do and you're likely to run into some merge conflicts if you don't do it right.

Reply:
P.S., I still see `<style scoped>` in this CSS file.